sbt/src/sbt-test/tests/bak/test

```diff
@@ -0,0 +1,3 @@
+> p1/t;clean;p1/t;clean;p1/t;clean
+$ sleep 100
```

Comment:
What is the purpose of the sleep?

Reply:
It's a voodoo sleep.

It was very difficult on my machine reliably. Using https://github.com/retronym/sbt-classes-bak-repro stand-alone (as opposed to scripted) often took several tries. "sleep 100" emulates my human pause, which to be honest I am not sure if it has any effect at all. Prior to the fix, I was able to get the scripted to fail I think once in 3 or 4 times.